x-pack/plugins/security_solution/public/hosts/pages/details/utils.ts

```diff
@@ -38,7 +41,7 @@ export const getBreadcrumbs = (
     {
       text: i18n.PAGE_TITLE,
       href: getUrlForApp(APP_ID, {
-        path: !isEmpty(search[0]) ? search[0] : '',
+        path: getTabsOnHostsUrl(HostsTableType.hosts, !isEmpty(search[0]) ? search[0] : ''),
```

Comment:
I think we can do 
```getTabsOnHostsUrl(HostsTableType.hosts, search[0])```
as the empty is handled by `appendSearch` in `getTabsOnHostsUrl`